sdk/storage/azure-storage-file-datalake/azure/storage/filedatalake/_serialize.py

```diff
@@ -43,6 +43,16 @@ def get_api_version(kwargs):
     return api_version or _SUPPORTED_API_VERSIONS[-1]
 
 
+def compare_api_versions(version1: str, version2: str) -> int:
+    v1 = _SUPPORTED_API_VERSIONS.index(version1)
+    v2 = _SUPPORTED_API_VERSIONS.index(version2)
```

Comment:
Can we add some inline comments to the `_SUPPORTED_API_VERSIONS` list to say that the list must be get in chronological order?